Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -52,36 +52,39 @@ bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) con
 {
     QStringList sections = line.split(BATDATA_DELIMITER);
 
-    if (sections.length() < COLUMNS)  {
+    if (sections.length() < COLUMNS)
+    {
         qDebug() << "sections length is less than 3";
         return false;
-    }   else if (sections.length() > COLUMNS)   {
+    }
+
+    else if (sections.length() > COLUMNS)
+    {
         qDebug() << "sections length is greater than 3";
         return false;
     }
 
     QString timeString = sections.at(0);
     batteryData.time = QTime::fromString(timeString, STRING_TIME_FORMAT);
 
-    bool double_conversion;
+    bool doubleConversionV;
+    bool doubleConversionC;
 
-    batteryData.voltage = sections.at(1).toDouble(&double_conversion);
+    batteryData.voltage = sections.at(1).toDouble(&doubleConversionV);
 
-    if (double_conversion == false) {
-        qDebug() << "double_conversion unsuccessful";
-        return false;
-}
-    batteryData.current = sections.at(2).toDouble(&double_conversion);
+    batteryData.current = sections.at(2).toDouble(&doubleConversionC);
 
-    if (double_conversion == false) {
+    if (doubleConversionV == false || doubleConversionC == false)
```

Comment:
Going with what I said earlier, can you guess what the equivalent statement would be?